client/rest/http.go

```diff
@@ -35,6 +35,9 @@ import (
 func NewHTTPClient(proxyUrl string) (*http.Client, error) {
 	transport := http.DefaultTransport.(*http.Transport).Clone()
 	transport.MaxConnsPerHost = 200
+	transport.MaxIdleConnsPerHost = 200
+	transport.MaxIdleConns = 200
```

Comment:
Setting MaxIdleConns to the same number as MaxConnsPerHost does technically bring potential problems with a single host hogging all the connections and leaving other hosts with no connections to pick up. While this _may not_ be a huge concern with the current architecture, it is a potential source of future issues.